src/apcups.c

```diff
@@ -436,8 +434,19 @@ static int apcups_read(void) {
   return 0;
 } /* apcups_read */
 
+static int apcups_init(void) {
+  if (conf_node == NULL)
+    conf_node = APCUPS_DEFAULT_NODE;
+
+  if (conf_service == NULL)
+    conf_service = APCUPS_DEFAULT_SERVICE;
+
+  return 0;
+} /* apcups_init */
+
 void module_register(void) {
   plugin_register_complex_config("apcups", apcups_config);
+  plugin_register_init("apcups", apcups_init);
```

Comment:
The goal is to allow plugin use without without defining an explicit configuration.
Then there will be `LoadPlugin apcups` only, without `<Plugin apcups>` block  in config file.
The `apcups_config()` will not be called at all in that case.